modules/install-sno-ibm-z-lpar.adoc

```diff
@@ -0,0 +1,227 @@
++
+[source,terminal]
+----
+$ OCP_VERSION=<ocp_version> <1>
+----
++
+<1> Replace `<ocp_version>` with the current version, for example, `latest-{product-version}`
+
+. Set the host architecture by running the following command:
++
+[source,terminal]
+----
+$ ARCH=<architecture> <1>
+----
+<1> Replace `<architecture>` with the target host architecture `s390x`.
+
+. Download the {product-title} client (`oc`) and make it available for use by entering the following commands:
++
+[source,terminal]
+----
+$ curl -k https://mirror.openshift.com/pub/openshift-v4/${ARCH}/clients/ocp/${OCP_VERSION}/openshift-client-linux.tar.gz -o oc.tar.gz
+----
++
+[source,terminal]
+----
+$ tar zxvf oc.tar.gz
+----
++
+[source,terminal]
+----
+$ chmod +x oc
+----
+
+. Download the {product-title} installer and make it available for use by entering the following commands:
++
+[source,terminal]
+----
+$ curl -k https://mirror.openshift.com/pub/openshift-v4/${ARCH}/clients/ocp/${OCP_VERSION}/openshift-install-linux.tar.gz -o openshift-install-linux.tar.gz
+----
++
+[source,terminal]
+----
+$ tar zxvf openshift-install-linux.tar.gz
+----
++
+[source,terminal]
+----
+$ chmod +x openshift-install
+----
+
+. Prepare the `install-config.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: <domain> <1>
+compute:
+- name: worker
+  replicas: 0 <2>
+controlPlane:
+  name: master
+  replicas: 1 <3>
+metadata:
+  name: <name> <4>
+networking: <5>
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16 <6>
+  networkType: OVNKubernetes
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  none: {}
+pullSecret: '<pull_secret>' <7>
+sshKey: |
+  <ssh_key> <8>
+----
+<1> Add the cluster domain name.
+<2> Set the `compute` replicas to `0`. This makes the control plane node schedulable.
+<3> Set the `controlPlane` replicas to `1`. In conjunction with the previous `compute` setting, this setting ensures the cluster runs on a single node.
+<4> Set the `metadata` name to the cluster name.
+<5> Set the `networking` details. OVN-Kubernetes is the only allowed network plugin type for single-node clusters.
+<6> Set the `cidr` value to match the subnet of the {sno} cluster.
+<7> Copy the {cluster-manager-url-pull} and add the contents to this configuration setting.
+<8> Add the public SSH key from the administration host so that you can log in to the cluster after installation.
+
+. Generate {product-title} assets by running the following commands:
++
+[source,terminal]
+----
+$ mkdir ocp
+----
++
+[source,terminal]
+----
+$ cp install-config.yaml ocp
+----
+
+. Change to the directory that contains the {product-title} installation program and generate the Kubernetes manifests for the cluster:
++
+[source,terminal]
+----
+$ ./openshift-install create manifests --dir <installation_directory> <1>
+----
++
+<1> For `<installation_directory>`, specify the installation directory that
+contains the `install-config.yaml` file you created.
```

Comment:
You can remove the line break and include the full sentence on line 123, unless there was a specific reason to add a line :)